### ISSUE TITLE
[Bug] java.lang.NullPointerException when opening new cron job dialog for an invalid Magento 2 module

### ISSUE THREAD

**Issue by rodzappa** [magento/magento2-phpstorm-plugin#825]
**Describe the bug** (*)

```
java.lang.NullPointerException
	at com.magento.idea.magento2plugin.actions.generation.dialog.NewCronjobDialog.suggestCronjobName(NewCronjobDialog.java:274)
	at com.magento.idea.magento2plugin.actions.generation.dialog.NewCronjobDialog.<init>(NewCronjobDialog.java:166)
	at com.magento.idea.magento2plugin.actions.generation.dialog.NewCronjobDialog.open(NewCronjobDialog.java:207)
	at com.magento.idea.magento2plugin.actions.generation.NewCronjobAction.actionPerformed(NewCronjobAction.java:48)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.lambda$performActionDumbAwareWithCallbacks$4(ActionUtil.java:244)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performDumbAwareWithCallbacks(ActionUtil.java:265)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performActionDumbAwareWithCallbacks(ActionUtil.java:244)
	at com.intellij.ide.actions.GotoActionAction.performActionImpl(GotoActionAction.java:111)
	at com.intellij.ide.actions.GotoActionAction.lambda$performAction$2(GotoActionAction.java:84)
	at com.intellij.openapi.application.TransactionGuardImpl.runWithWritingAllowed(TransactionGuardImpl.java:214)
	at com.intellij.openapi.application.TransactionGuardImpl.access$200(TransactionGuardImpl.java:21)
	at com.intellij.openapi.application.TransactionGuardImpl$2.run(TransactionGuardImpl.java:196)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:805)
	at com.intellij.openapi.application.impl.ApplicationImpl.lambda$invokeLater$4(ApplicationImpl.java:348)
	at com.intellij.openapi.application.impl.FlushQueue.doRun(FlushQueue.java:82)
	at com.intellij.openapi.application.impl.FlushQueue.runNextEvent(FlushQueue.java:131)
	at com.intellij.openapi.application.impl.FlushQueue.flushNow(FlushQueue.java:47)
	at com.intellij.openapi.application.impl.FlushQueue$FlushNow.run(FlushQueue.java:187)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:313)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:776)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:727)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:746)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:891)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:760)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$6(IdeEventQueue.java:447)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:818)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$7(IdeEventQueue.java:446)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:805)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:492)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)

```

**To Reproduce** (*)

**Expected behavior** (*)

**Screenshots**

**Please complete the following information:** (*)

 - OS: Linux 5.15.5-76051505-generic
 - PhpStorm/Intellij version: 2021.3
 - Plugin Version: 4.2.0

**Additional context**

**Comment by bohdan-harniuk** [comment]
Hello, @rodzappa!

Thank you for your report!

**Comment by bohdan-harniuk** [comment]
This issue is reproduced successfully:

<img width="827" alt="Screenshot 2021-12-17 at 16 05 19" src="https://user-images.githubusercontent.com/31848341/146556174-e4d7d2b1-0dc4-4aef-8cf5-d9302bd363fd.png">

**Comment by bohdan-harniuk** [comment]
Hello, @rodzappa !

We've fixed this issue.
The bug fix will be released in the upcoming 4.2.2 version.

Thank you,
Bohdan
